src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2661,6 +2661,507 @@ Note: `aparse()` is case-insensitive.
     </CollapserGroup>
   </Collapser>
 
+  <Collapser
+    className="freq-link"
+    id="func-toTimestamp"
+    title={<InlineCode>toTimestamp(timestamp[, pattern [, timezone]])</InlineCode>}
+  >
+    Use the `toTimestamp()` function to parse a timestamp in epoch milliseconds from a datetime string.
+
+    `toTimestamp()` takes the following arguments:
+
+    * datestring - A datetime string to be translated into a timestamp (epoch milliseconds). This can be a string attribute or a string literal in quotes.
+    * pattern - An optional date time pattern used to try and parse the datestring parameter. See the [Patterns for toTimestamp and toDatetime](#toTimestamp-toDatetime-patterns) section below.
+      * This must be a constant string value and will default to `yyyy-MM-dd'T'HH:mm:ss[.SSS][XXX]` if a pattern is not provided.
+    * timezone - An optional timezone value that is used to interpret the datestring parameter(ex. PST).
+      * This must be a constant string value and will default to UTC, or the value provided in `WITH TIMEZONE` if available.
+
+
+    Alias: `fromDatetime()` is an alternative name for `toTimestamp()` function.
+
+    <Callout variant="tip">
+      If the string found does not match the given pattern, it will return `null`. If you happen to have datetime strings in a variety of patterns, you can coalesce results by leveraging `OR` to cascade until one of the values is non-null. You can also use optional pattern segments. The default pattern uses square brackets to make the milliseconds and zone-offset parts optional.
+    </Callout>
+
+    <CollapserGroup>
+      <Collapser id="partial-datetime-interpretation" title="Partial datetime interpretation when parsing datetime strings">
+        <table>
+          <thead>
+          <tr>
+            <th>
+              Scenario
+            </th>
+            <th>
+              Detail
+            </th>
+            <th>
+              Sample pattern
+            </th>
+            <th>
+              Sample datetime
+            </th>
+            <th>
+              Resolves to
+            </th>
+          </tr>
+          </thead>
+
+          <tbody>
+          <tr>
+            <td>Missing time zone</td>
+            <td>Will use timezone argument, or WITH TIMEZONE value. Defaults to UTC.</td>
+            <td>yyyy-MM-dd HH:mm:ss.SSS</td>
+            <td>2023-10-18 15:27:03.123</td>
+            <td>2023-10-18T15:27:03.123Z</td>
+          </tr>
+          <tr>
+            <td>Missing datetime field</td>
+            <td>
+              Any missing time field will be replaced with 0.<br/>
+              Any missing date field will be replaced with 1.<br/>
```

Comment:
The line breaks here made it look to me as if these were scenarios that applied to the examples on the same line, and I was puzzled that e.g. "yyyy-MM-dd HH:mm:ss" didn't appear to have a missing time field.  What do you think about omitting these two line breaks?

Reply:
This makes sense. In the initial Nerdlife post I believe the line breaks were there but I've confirmed they do not apply to the examples on the same line.